Code/GraphMol/FileParsers/PDBParser.cpp

```diff
@@ -449,6 +449,23 @@ static void StandardPDBResidueChirality(RWMol *mol) {
   }
 }
 
+void BasicPDBCleanup(RWMol &mol) {
+  ROMol::VERTEX_ITER atBegin, atEnd;
+  boost::tie(atBegin, atEnd) = mol.getVertices();
```

Comment:
For my education, why do you use `getVertices` and not an `AtomIterator`? 

And to be picky, `atBegin` points only at the start of the iteration to the begin of the atom list.